_config/extensions.yml

```diff
@@ -8,6 +8,11 @@ SilverStripe\Assets\File:
   extensions:
     - BasicFieldsTestFileExtension
 
+SilverStripe\FrameworkTest\Model\TestBlogPost:
+  extensions:
+    - DNADesign\Elemental\Extensions\ElementalAreasExtension
+    - SilverStripe\FrameworkTest\Extension\TestBlogPostExtension
```

Comment:
This will cause errors for other tests which don't include the elemental module. Include it instead inside the below section which already checks if the elemental extension class exists (it should probably also be swapped from `classexists` to `moduleexists: 'dnadesign/silverstripe-elemental'`)